Azure-Samples/bc-community-samples
# Submission for the ebc-forms-reactjs bounty.

> QUICK VIEW
>
> Quickly view this PR in my fork's [README.md](https://github.com/JakubNer/bc-community-samples/tree/master/ebc-forms-reactjs)
>
> Quickly see the demo application from my [fork's GitHub pages](https://jakubner.github.io/bc-community-samples/ebc-forms-reactjs/app/index.html)

## Purpose
Submission for the ebc-forms-reactjs bounty. 

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install -g http-server
http-server app
```

Open browser at http://localhost:8080 to see react app.

Open form as per README.md

Read the README.md quick starts.

* Test the code

```
Play around as per README.md
```
